samza-core/src/main/java/org/apache/samza/metrics/reporter/MetricsHeader.java

```diff
@@ -21,13 +21,15 @@
 import java.util.HashMap;
 import java.util.Map;
 import java.util.Objects;
+import java.util.Optional;
 
 
 public class MetricsHeader {
   private static final String JOB_NAME = "job-name";
   private static final String JOB_ID = "job-id";
   private static final String CONTAINER_NAME = "container-name";
   private static final String EXEC_ENV_CONTAINER_ID = "exec-env-container-id";
+  private static final String EXEC_ENV_ATTEMPT_ID = "exec-env-attempt-id";
```

Comment:
nit: should we call it 
exec-env-job-attempt-id
or 
job-attempt-id

Reply:
Good suggestion. I like using `samzaEpochId`